pyproject.toml

```diff
@@ -26,14 +26,14 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.11"
-numpy = "1.23.5"
-scipy = "1.8.0"
-numba = "0.56.4"
-thewalrus = "0.19.0"
-rich = "10.15.1"
-matplotlib = "3.5.0"
-tqdm = "^4.65.0"
-ray = { version = "2.5.0", extras = ["tune"], optional = true }
+grpcio = "1.60.0"
+numpy = "^1.23.5"
+scipy = "^1.8.0"
+numba = "^0.56.4"
+thewalrus = "~0.19.0"
```

Comment:
Why not bumping the walrus version? Is there any breaking change in the new releases?

Reply:
This opens up a bigger conversation, which is that I think it would be helpful to have every dependency unpinned.
With a good nightly build one can learn about bugs that come with new versions and fix them right away.

This is the only way to make sure that people doing `pip install mrmustard` on an existing environment can have it working